index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-gH2yIJqKdNHPEq0n4Mqa/HGKIhSkIHeL5AyhkYV8i59U5AR6csBvApHHNl/vI1Bx" crossorigin="anonymous">
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+
+  <header id="header">
+    <nav id="nav-bar" class="navbar navbar-expand-lg navbar-light" style=" background-color: #e3f2fd;">
+      <a class="navbar-brand" href="#">Landing Page</a>
+      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent"
+        aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+      </button>
+
+      <div class="collapse navbar-collapse" id="navbarSupportedContent">
+        <ul class="navbar-nav mr-auto">
+          <li class="nav-item active">
+            <a class="nav-link" href="#">Home</span></a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#form">Link</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#video">Link</a>
+          </li>
+        </ul>
+      </div>
+    </nav>
+    <img id="header-img" src="https://img.techentice.com/media/2019/10/embedded.jpg">
+  </header>
+  <form id="form" action="https://www.freecodecamp.com/email-submit">
+    <input id="email" name="email" type="email" placeholder="type your email">
+    <input id="submit" type="submit">
+  </form>
+  <video id="video" width="640" height="480" controls>
+    <source src="https://www.youtube.com/watch?v=oPn_adlC1Q0" type="video/mp4">
+  </video>
+  <div class="flex-box">
+  </div>
+
+  <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
```

Comment:
don't use unused code as a comment.

Reply:
- [ ] 🛑  kindly try always to include an explanation, a "why", for each requested change. For example:

> - **_This is a well-explained request_**: Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible
> - _**This request lacks a "why":**_ Why don't you use an article tag here instead of a div?



*******************************

- [ ] 🛑 kindly try to phrase the comments politely, as suggestions instead of orders.
For example:

> - This is a polite suggestion: "I think it would be better to delete the comments"
> - 
> - This reads as an order: " So delete the comments"